inlinino/gui.py

```diff
@@ -1290,8 +1306,11 @@ def start(self, instrument_index=None):
             try:
                 instrument_class = {'generic': Instrument, 'acs': ACS, 'apogee': ApogeeQuantumSensor,
                                     'dataq': DATAQ, 'hyperbb': HyperBB, 'hypernav': HyperNav,
-                                    'lisst': LISST, 'nmea': NMEA, 'ontrak': Ontrak, 'satlantic': Satlantic,
-                                    'sunav1': SunaV1, 'sunav2': SunaV2, 'taratsg': TaraTSG}
+                                    'lisst': LISST, 'nmea': NMEA,
+                                    #'ontrak': Ontrak,
```

Comment:
Ontrak can now be uncommented here, following discussion on handling ImportError in ontrak specific to Windows.

Reply:
Missed this one. Thanks.